backend/api_calls.py

```diff
@@ -819,11 +816,24 @@ def AddRewards(request, firebase):
 
   for i in range(request['count']):
     reward = 'reward-%s-%s' % (reward_seed, RandomWords(3))
-    results.append({reward: firebase.put('/rewards', reward, reward_data)})
+    results.append(AddRewardToDb(firebase, reward))
 
   return results
 
 
+def AddRewardToDb(firebase, reward):
+  """Put a new reward into the DB.
+
+  Firebase entries:
+    /rewards/%(rewardId)
+    /rewardCategories/%(rcID)/rewards/%(rewardId)
+  """
+  firebase.put('/rewards', reward, {'playerId': '', 'a': True})
+  reward_category = 'rewardCategory-%s' % reward.split('-')[1]
```

Comment:
reward.split('-', maxsplit=1) to enforce one split?

(Or maybe just reward[len('reward-'):])?